README.md

```diff
@@ -71,6 +72,23 @@ Or, if you've been mimicked a little harder,
 <img alt="some worse code"
      src="https://cloud.githubusercontent.com/assets/1236420/10564914/f7963ae4-7591-11e5-9b45-f123e42b22f4.png" />
 
+### Installation
```

Comment:
Actually, it appears I've already done this in the merged version. I surprised myself by my prescience here :smile:.